neoassistant/commands.py

```diff
@@ -241,10 +241,22 @@ def execute(self, assistant: Assistant, args):
 
 class ShowBirthdaysCommand(Command):
     def __init__(self):
-        super().__init__("show-birthdays", "Show all birthdays per next 7 days.")
+        super().__init__(
+            "show-birthdays", 
+            "Show all birthdays per the next specified number of days," + 
+            " if days are not specified show per next 7 days."
+            )
 
-    def execute(self, assistant: Assistant, _):
-        return assistant.contact_book.get_birthdays_per_week()
+    def execute(self, assistant: Assistant, args):
+        days_delta = 7
+        if args:
```

Comment:
Remove `if` check because it is always true, if there is not argument the list will be empty but it is a list, not a None

Reply:
When I remove `if` check, the show-birthdays command without parameters raise an error.
`Enter a command: show-birthdays  
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "E:\Master Woolf\goitneo-python-final-project-group-11\.venv\Scripts\neoassistant.exe\__main__.py", line 7, in <module>     
  File "E:\Master Woolf\goitneo-python-final-project-group-11\neoassistant\__main__.py", line 27, in main
    print(f"\n{command_object.execute(neoassistant, args)}\n")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\Master Woolf\goitneo-python-final-project-group-11\neoassistant\commands.py", line 254, in execute
    days_delta = int(args[0])
                     ~~~~^^^
IndexError: list index out of range`